packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -1211,7 +1211,18 @@ export class WalletService {
         name: 'tokenAddresses',
         isValid(value) {
           return (
-            _.isArray(value) && value.every(x => x.length < 45 && x.indexOf('0x') == 0)
+            _.isArray(value) && value.every(x => {
+              try {
+                Validation.validateAddress(
+                  'eth',
+                  'mainnet',
+                  x,
+                );
```

Comment:
This function will not throw if its just an invalid address, it only throws if theres a real error. 

return the function instead:
```
return Validation.validateAddress(
                  'eth',
                  'mainnet',
                  x,
                );
```

Reply:
ok, then we have an error on _validateAddr, fixing.